traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -274,16 +274,16 @@ func GetCDNDomainFromName(tx *sql.Tx, cdnName tc.CDNName) (string, bool, error)
 	return domain, true, nil
 }
 
-// ServerExists returns true if the server exists.
-func ServerExists(serverName string, tx *sql.Tx) (bool, error) {
-	id := 0
+// GetServerIDFromName gets server id from a given name
+func GetServerIDFromName(serverName string, tx *sql.Tx) (int64, bool, error) {
+	id := int64(0)
```

Comment:
The type of a `tc.Server.ID` field is `int`, so this should return that to avoid casting when passing its output into the object.

Reply:
good call was attempting to just match the GetServerNameFromID id type. Think I should change it there as well to just int?

Reply:
Sure, why not, that's what tests are for, huh? Personally I think we ought to be using `uint` or `uint64` but that's a sweeping change to the codebase.